lib/grape/jsonapi/parameters/filter.rb

```diff
@@ -91,18 +91,22 @@ def validate!
 
           # rubocop:disable Metrics/MethodLength
           def query_for(model)
-            model.tap do |query|
-              filters do |key, op, value|
-                case op
-                when OP_EQ
-                  query.where(key => value)
-                when OP_IN
-                  query.in(key => value)
-                else
-                  query.send(op, key => value)
-                end
-              end
+            result = nil
```

Comment:
please rewrite this to use reduce, as in filters.reduce(model)
look at this history of this method (before my branch) for an example